app/login/template.hbs

```diff
@@ -0,0 +1,4 @@
+{{input type = "text" value = userEmail placeholder = "Email"}} <br>
```

Comment:
If you created a `login-form` component, when a user clicks "Login", the component can send the username and password an action. This would prevent you from having to get properties from the controller in the route.

```hbs
{{login-form action="login"}}
```

```js
login(username, password) {
   //do stuff
}
```